packages/lib/constants.ts

```diff
@@ -17,7 +18,7 @@ export const WEBAPP_URL =
 
 // OAuth needs to have HTTPS(which is not generally setup locally) and a valid tld(*.local isn't a valid tld)
 // So for development purpose, we would stick to localhost only
-export const WEBAPP_URL_FOR_OAUTH = IS_PRODUCTION ? WEBAPP_URL : "http://localhost:3000";
+export const WEBAPP_URL_FOR_OAUTH = IS_PRODUCTION || IS_DEV ? WEBAPP_URL : "http://localhost:3000";
```

Comment:
Confirmed in a replt that this will return true if either `IS_PRODUCTION` or `IS_DEV` is true.